Sources/SwifterSwift/UIKit/UILabelExtensions.swift

```diff
@@ -9,7 +9,56 @@
 #if canImport(UIKit) && !os(watchOS)
 import UIKit
 
+// MARK: - Properties
+public extension UILabel {
+
+    /// SwifterSwift: Required height for a label
+    var requiredHeight: CGFloat {
+        let label = UILabel(frame: CGRect(x: 0, y: 0, width: frame.width, height: CGFloat.greatestFiniteMagnitude))
+        label.numberOfLines = 0
+        label.lineBreakMode = NSLineBreakMode.byWordWrapping
+        label.font = font
+        label.text = text
+        label.attributedText = attributedText
+        label.sizeToFit()
+        return label.frame.height
+    }
+}
+
 // MARK: - Methods
+public extension UILabel {
+
+    /// SwifterSwift:  Calculate and return the size that best fits an attributed string, given the specified constraints on size
+    ///
+    ///   let yourAttributedString = ...
+    ///   let yourConstraintSize = ...
+    ///   let size = UILabel.size(thatFitsAttributedString: yourAttributedString, withConstraints: yourConstraintSize)
+    ///
+    /// - Parameters:
+    ///   - attributedString: The attributed string.
+    ///   - size: The maximum dimensions used to calculate size.
+    /// - Returns: The size that fits the attributed string within the specified constraints.
+    @available(iOS 10, *)
```

Comment:
The minimum target we have is already 10 I think, so I think we don't need this :)

Reply:
Fixed now.